### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by junhocho** [titu1994/Super-Resolution-using-Generative-Adversarial-Networks#1]
Hey, great work.

Did you measure your SRResNet PSNR/SSIM ? 
And compare with the paper?

**Comment by titu1994** [comment]
@junhocho SRResNet would be just the generative model with MSE loss right? Haven't tested that. I can add that in some time.

**Comment by titu1994** [comment]
@junhocho Just pushed a bunch of code in the "tests" folder, which has a benchmark script. To get the validation images, open the "set5", "set14" and "bsd100" folders and run the download_*.py scripts to get the validation images.

After that, run benchmark_test.py to test against all the datasets. 

For now, because of the upscaling error (blurry, RGB grids):
**Set5 :** Average PSNR of Set5 validation images :  22.1211430348
**Set14 :** Average PSNR of Set5 validation images :  20.3971611357
**BSD100 :** Average PSNR of BSD100 validation images :  20.9544390316

**Comment by titu1994** [comment]
I pushed a few changes to the benchmarking code to upsample at higher resolutions. This seems to improve results somewhat. 

New scores:
**Set5 :** Average PSNR of Set5 validation images : **25.9486824127** 
**Set14 :** Average PSNR of Set5 validation images : **22.7537558933**
**BSD100 :** Average PSNR of BSD100 validation images : **23.9422518821**

This poses the question : if the testing is improved somewhat due to larger test image sizes, then perhaps the image size trained by the twitter team (96x96) was primarily to avoid the pixel grids.

I will test by pretraining a model with 96x96 sized images. This is still possible since it is the discriminator model which adds a tremendous number of parameters (303 million~) at 96x96 image size, but just SR ResNet has barely 1.2 million parameters. The larger image size may be the key to correcting the pixel grid formation.
